apps/cbportal/base/CBGrid/Cloud Spaces/Cloud Space.wiki

```diff
@@ -72,14 +74,21 @@ h2. Cloud Space: ${name}
 
 {% endif %}
 
-- display: Delete Cloud Space
+- display: Delete cloud Space
```

Comment:
Titles are uppercased why did you lower this? PLease uppercase again make sure to refer to it correctly from any actions